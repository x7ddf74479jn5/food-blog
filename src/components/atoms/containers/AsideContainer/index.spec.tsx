import renderer from "react-test-renderer";

import { AsideContainer } from ".";

describe("components/atoms/containers/AsideContainer", () => {
  it("snapshot", () => {
    const treeLeft = renderer.create(<AsideContainer side="left">children</AsideContainer>).toJSON();
    expect(treeLeft).toMatchSnapshot();
    const treeRight = renderer.create(<AsideContainer side="right">children</AsideContainer>).toJSON();
    expect(treeRight).toMatchSnapshot();
  });
});
