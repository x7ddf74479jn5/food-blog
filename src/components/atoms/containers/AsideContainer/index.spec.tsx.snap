// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`components/atoms/containers/AsideContainer snapshot 1`] = `
<aside
  className="flex flex-col flex-1 order-2 md:order-1  gap-y-8 items-center md:mt-16 w-full  min-w-max lg:w-1/3 h-auto undefined"
>
  <div
    className="flex flex-col h-full"
  >
    <div
      className="flex md:sticky md:top-16"
    >
      children
    </div>
  </div>
</aside>
`;

exports[`components/atoms/containers/AsideContainer snapshot 2`] = `
<aside
  className="flex flex-col flex-1 order-3 gap-y-8 items-center md:mt-16 w-full  min-w-max lg:w-1/3 h-auto undefined"
>
  <div
    className="flex flex-col h-full"
  >
    <div
      className="flex md:sticky md:top-16"
    >
      children
    </div>
  </div>
</aside>
`;
